### ISSUE TITLE
Cantarell font not rendering properly: looking stretched or thin

### ISSUE THREAD

**Issue by piktonus97m** [pop-os/gtk-theme#571]
**Describe the problem:**

Hello team, as you can see in the screenshot, for some reason the Cantarell font is looking very stretchy or super think is some parts of the interface, even though changing the font size, it still keeps the same behavior.

![image](https://user-images.githubusercontent.com/37163970/152462067-eda93fe4-e36c-41cb-afb9-1c33d7c7d406.png)

![image](https://user-images.githubusercontent.com/37163970/152463156-27b8e478-e758-4d36-8cdb-d6855bb21811.png)

![image](https://user-images.githubusercontent.com/37163970/152463275-e54e69d9-e00a-4bf8-8c3d-1ac009fd45d6.png)

**Steps to reproduce:**
1. Download and install the [Cantarell](https://cantarell.gnome.org/) font.
2. On Gnome-Tweaks change the font to Cantarell.

**Distribution:** PRETTY_NAME="Pop!_OS 21.10"

**Pop version:** Installed: 5.4.5~ 1641939568 ~21.10 ~bf3a4f4

```
Pop Gtk theme - Pop Gtk theme used by System76

          ID: org.gtk.Gtk3theme.Pop
         Ref: runtime/org.gtk.Gtk3theme.Pop/x86_64/3.22
        Arch: x86_64
      Branch: 3.22
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 1.1 MB

      Commit: 1eac06f6a58047fd6717a13f12b744ea59e6f9829fed02d856a5efb998b318a5
      Parent: bf0014cc176fe2a66eb69045bc3ed80663e09240dda1db97b5394ba3a501bea3
     Subject: add flathub.json (84e0033a)
        Date: 2020-11-17 21:00:45 +0000
```

**Comment by isantop** [comment]
I don't think this is an issue with the theme, as the theme does not control font rendering. Do you still get this issue using a different theme selected?

**Comment by piktonus97m** [comment]
Hello @isantop, that is correct using another theme keeps showing the same font rendering. If the theme is not the case; where I can reference this issue? Thanks in advanced.
